woocommerce/woocommerce-gutenberg-products-block
# Footer with 2 Menus Dark pattern: Fix the Site Title color contrast in TT4.

## What

Fixes #11293 

## Why

This PR fixes the Site Title color contrast issue in the **Footer with 2 Menus Dark** pattern with **Twenty Twenty-Four**.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Activate the **Twenty Twenty-Four** theme.
2. Create a new page.
3. Add the **Footer with 2 Menus Dark** pattern.
4. Verify that the Site Title is visible and has correct color contrast both in the editor and the frontend.
5. Enable the **Twenty Twenty-Three** theme and repeat all steps to make sure there are no regressions.


## Screenshots or screencast



### Before

| Editor | Front end |
| ------ | ----- |
|<img width="1213" alt="Edit_Page_“Footer_with_2_Menus_Dark”_‹_newproductgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/b3d5bdf4-1407-466b-a4f9-009db2b0a8d4">|<img width="1215" alt="Cursor_i_Footer_with_2_Menus_Dark_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/e44e5091-a944-46bb-b553-aec56e4bad13">|


### After

| Editor | Front end |
| ------ | ----- |
|<img width="1219" alt="Cursor_i_Edit_Page_“Footer_with_2_Menus_Dark”_‹_newproductgallery_—_WordPress" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/f2eacf22-03bb-4d99-815d-c2d2aa088d56">|<img width="1216" alt="Cursor_i_Footer_with_2_Menus_Dark_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/2fe73bb6-62b5-44b7-ac2f-95ae56bceabe">|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Footer with 2 Menus Dark pattern: Fix the Site Title color contrast in Twenty Twenty-Four.
